### PR TITLE
mariadb 10.1.11

### PR DESCRIPTION
**Pull request by makigumo**

### CODE REVIEW

File: Library/Formula/mariadb.rb [hunk 1]
```diff
@@ -146,6 +139,8 @@ def install
         s.gsub!("$(dirname $0)/wsrep_sst_common", "#{libexec}/wsrep_sst_common")
       end
     end
+    # remove non-executable from bin
+    rm bin+"maria_add_gis_sp.sql"
```
Comment (DomT4) [inline Library/Formula/mariadb.rb]: We sure this isn't important?

Comment (makigumo) [inline Library/Formula/mariadb.rb]: I checked for usages of this file and found none. There is still a copy of this file located in ```share/mysql/```.
```mysql_install_db``` only uses ```maria_add_gis_sp_bootstrap.sql``` which has the exact same functionality.

Comment (DomT4) [inline Library/Formula/mariadb.rb]: Alright, I'll trust ya :wink:

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/9aa38c0722b0f25305fc82611c2e92aa7442d020. Thank you for another contribution to Homebrew @makigumo! :smiley_cat:

**Pull request closed without merging**